transport-msg.tex

```diff
@@ -737,52 +738,47 @@ \subsubsection{Overview}
 \hline
 \msgref{RESET_VQUEUE}      & 0xB  & Driver \\
 \hline
-\msgref{EVENT_CONFIG}      & 0x20 & Device \\
+\msgref{EVENT_CONFIG0}     & 0x40 & Device \\
+\hline
+\msgref{EVENT_CONFIG}      & 0x41 & Device \\
 \hline
-\msgref{EVENT_AVAIL}       & 0x21 & Driver \\
+\msgref{EVENT_AVAIL}       & 0x42 & Driver \\
 \hline
-\msgref{EVENT_USED}        & 0x22 & Device \\
+\msgref{EVENT_USED}        & 0x43 & Device \\
 \hline
 \end{tabular}
 
+Transport message IDs 0x00 to 0x3F are used for messages that require a response
+and IDs 0x40 to 0x7F are used for messages that have no response.  Transport
+message IDs 0x80 and above are reserved by this specification.
+
 \paragraph{Mandatory Transport Messages}
 For a virtio-msg device to be fully operational, the following messages
 \textbf{MUST} be supported:
 \begin{itemize}
-  \item \msgref{ERROR}
   \item \msgref{GET_DEVICE_INFO}
   \item \msgref{GET_FEATURES} and \msgref{SET_FEATURES}
```

Comment:
needs fixing to GET_DEVICE_FEATURES and SET_DRIVER_FEATURES (other occurrences in the doc)

Reply:
humm, I did do a global search ... but I guess I failed somehow